### PR TITLE
add periodic job to run full e2e tests in cloud-provider-gcp with latest k/k libraries

### PR DESCRIPTION
**Pull request by andrewsykim**

### CODE REVIEW

File: config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml [hunk 1]
```diff
@@ -127,6 +127,72 @@ periodics:
           echo "TEST_PACKAGE_VERSION - Falling back to v1.25.0";
         fi;
         kubetest2 gce -v 2 --repo-root $REPO_ROOT --build --up --down --test=ginkgo --node-size n1-standard-4 --master-size n1-standard-8 -- --test-package-version="${TEST_PACKAGE_VERSION}" --parallel=30 --test-args='--minStartupPods=8 --ginkgo.flakeAttempts=3' --skip-regex='\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]'
+
+- interval: 6h
+  cluster: k8s-infra-prow-build
+  name: ci-cloud-provider-gcp-e2e-latest-with-kubernetes-master
```
Comment (andrewsykim) [inline config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml:133]: This is identical to `ci-cloud-provider-gcp-e2e-latest` but adds a call to `e2e/add-kubernetes-to-workspace.sh;` right before calling kubetest2

**Comment by dims** [comment]
/approve

leaving `/lgtm` to @aojea

**Pull request merged**